src/py42/services/savedsearch.py

```diff
@@ -32,19 +32,21 @@ def get_by_id(self, search_id):
         uri = u"{}/{}".format(self._resource, search_id)
         return self._connection.get(uri)
 
-    def get_query(self, search_id):
+    def get_query(self, search_id, pg_num=None, pg_size=None):
```

Comment:
for consistency with other methods, this should be `page_size` and `page_num`. The base class is also looking for keyword arguments by these names.

Reply:
To clarify, is this `page_num` or `page_number`? `BaseQuery` is `page_number` that's why I ask.

Reply:
@DiscoRiver  You correct that it is `page_number` in the query classes... I have `page_num` in other parts of the code base though.  Perhaps we should make a plan to standardize these.

Reply:
Sounds good. I'll change it to `page_number` for now, but I can raise an issue for standardizing it (happy to do the work on that too).

Reply:
I am delighted at how much you are wanting to work on py42! Much appreciated. I think we should save the page num standardization for a greater release, probably like a 2.0, but if it is something you want to take on, I will try to make that happen when that time comes.